sesheta/common.py

```diff
@@ -92,3 +92,19 @@ def get_labels(pr) -> list:  # pragma: no cover
         _LOGGER.error(e)
 
     return labels
+
+
+def commit_was_successful_tested(commit, statuses):
```

Comment:
is this working with Zuul?

Reply:
yes, I guess as it started adding the tag to pr's.
for example https://github.com/thoth-station/cleanup-job/pull/158
I will ensure to verify it.